jvb/src/main/resources/reference.conf

```diff
@@ -229,6 +229,8 @@ videobridge {
   websockets {
     enabled=false
     server-id="default-id"
+    # Whether to use WebSocket compression (permessage-deflate)
+    compression = true
```

Comment:
In terms of what to call the parameter, you mean?

I was modeling it on the adjacent `tls` parameter, but I can use something else. (Probably `dash-separated` rather than `interCap` though.)

Reply:
Yes and yes, good point.

Reply:
As far as tls, I think that should probably be named `enable-tls` or something...so I'd say let's make this one better and we can fix that someday.